src/_getFilesAncestry.ts

```diff
@@ -0,0 +1,32 @@
+import { ContentRecord } from "@activeviam/activeui-sdk-5.0";
+
+function addChildren(
```

Comment:
I find `addChildren` misleading, as we're not adding the child nodes themselves:
```suggestion
function accumulateAncestryOfChildren(
```

Reply:
Good point, although I prefer `accumulateChildrenAncestors`, I find it easier to read, what do you think?

Reply:
I think a "child ancestor" could be confusing. But what about `accumulateAncestorsOfChildren` or just `accumulateAncestors` then ?
If we go for that, then we could align on it with `_getFileAncestors`

Reply:
As discussed, let's go for `accumulateAncestry` and keep `_getFilesAncestry`